.github/workflows/ci-linux.yml

```
@@ -53,6 +53,10 @@ jobs:
         run: |
           vendor/bin/phpcs --standard=PSR12 lib/php/*
 
+      - name: Run code static analysis
+        run: |
+          vendor/bin/phpstan analyze --ansi --memory-limit=2G --no-interaction --no-progress
```

I guess the lib is small enough to not result in memory limit issues.
```suggestion
          vendor/bin/phpstan analyze --ansi --no-interaction --no-progress
```